changelog/unreleased/update-web-8.0.0.md

```diff
@@ -0,0 +1,64 @@
+Enhancement: Update web to v8.0.0
```

Comment:
Although this is just an alpha version for `v8.0.0`, I decided to create the changelog item already. Future alpha version bumps can expand it of course. But I want to avoid having multiple changelog items for different `8.0.0-alpha` versions.

Reply:
What about stating the alpha version and then adjust it in the next releases?

Reply:
We can try again, last time it was a little chaotic and we eventually had 3 different changelogs for the Web bump 😄 

<img width="200" alt="image" src="https://github.com/owncloud/ocis/assets/50302941/4f8ceed2-3f85-44ba-b126-981fa45ec31f">